### ISSUE TITLE
3.3-Color-Clock-Joey Gaskin

### ISSUE THREAD

**Issue by joeygaskin** [TIY-GVL-FEE-2015-August/assignments#123]
#### Getting Started
  * [x] Create a new issue with tasks
  * [x] Create a new repository
  * [x] Create and commit `index.html`
  * [x] HTML Boilerplate
  * [x] Add structure using HTML

#### Make it Tick
  * [x] Create and commit a new file in called `main.js`
  * [x] Created and committed and **RAN** `main.js`
  * [x] `console.log` the current time on page load.
  * [x] `console.log` the current time every second.
  * [ ] Display the current time every second
  * [ ] Display the current time, padded with zeros to be exactly two digits
    long, every second.
  * [ ] `console.log` the percentage of a minute that the current seconds
    represents (e.g., if 30 seconds have elapsed, `console.log` 0.5)
  * [ ] Using the percent above, dynamically modify the length of the timer bar.
  * [ ] `console.log` a hexidecimal color that is based on the current second
    every second.
  * [ ] Dynamically update the background color of the page using the color.
  * [ ] Display the generated color's hex value on hover.
  * [ ] Mission Complete!

**Comment by joeygaskin** [comment]
Unfortunately, I was unable to complete this assignment. I wasn't able to figure out how to get started. I find that to be trend, at least for me anyway. If I knew how to organize JavaScript, I'm sure I could have produced something for this assignment. I googled a lot of different searches and even found clocks that attributed the results the assignment required, but I felt that all I had left to contribute was copying and pasting, which obviously isn't learning.

Today you went over this assignment in class, and I left with a bit more clarity but still left feeling stuck. So, I decided to use my time today to take several JavaScript courses on TreeHouse, and it's been a major help just breaking down simple syntax and learning how it all fits and interacts together. I really think [an hope] this will be helpful with the assignment that you gave for the weekend. However, this is yet another assignment that I will need to complete at a later time. 

- restless future junior dev  :-/
